load/load_distributor.go

```diff
@@ -1,12 +1,44 @@
 package load
 
-import "fmt"
+import (
+	"fmt"
+
+	consumer "github.com/AdityaMayukhSom/ruskin/consumer"
+)
+
+// Making a map of TopicId and LoadPartition
+// The map values have to be mapped to LoadPartition uid(will implement later)
+var TopicToPartitionMap = make(map[string]*LoadPartition)
```

Comment:
Do not make global variables. Each load distributor should have the map inside the structure.